src/components/CardArea/CardArea.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import { withStyles } from "@material-ui/core/styles";
+import PlayCard from "../PlayCard";
+import { Grid } from "@material-ui/core";
+
+const styles = theme => ({
+  root: {
+    minHeight: "100%",
+    maxHeight: "100%",
+    padding: "0vh",
+    margin: "0vh",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center"
+  }
+});
+
+class CardArea extends Component {
+  generateBoard = (board, clickAction) => {
+    let cards = [];
+    for (let i = 0; i < board.length; i++) {
+      //Inner loop to create children
+      cards.push(
+        <Grid item xs>
+          <PlayCard clickAction={clickAction} index={i} />{" "}
```

Comment:
le `{" "}` reste un peu louche

Reply:
yea its my auto linter that does that I dunno why